### PR TITLE
Batch subsystem configuration UI abstraction

### PR DESCRIPTION
**Pull request by zebrik**

### CODE REVIEW

**Comment by zebrik** [comment]
@rhatlapa please review

**Comment by zebrik** [comment]
I have force pushed with `WindowState`
@rhatlapa please review again

File: common/src/main/java/org/jboss/hal/testsuite/page/config/BatchPage.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+ */
+public class BatchPage extends ConfigurationPage implements Navigatable {
+
+    @Override
+    public void navigate() {
+        FinderNavigation navi;
+        if (ConfigUtils.isDomain()) {
+            navi = new FinderNavigation(browser, DomainConfigEntryPoint.class)
+                .addAddress(FinderNames.CONFIGURATION, FinderNames.PROFILES)
+                .addAddress(FinderNames.PROFILE, "full");
```
Comment (rhatlapa) [inline common/src/main/java/org/jboss/hal/testsuite/page/config/BatchPage.java]: Why aren't you loading the profile name from `org.jboss.hal.testsuite.util.ConfigUtils#getDefaultProfile` ?

**Comment by rhatlapa** [comment]
Just one minor issue, see comment above, otherwise looks good to me

**Comment by zebrik** [comment]
Force pushed selection of default profile. @rhatlapa please review once more time.

**Comment by rhatlapa** [comment]
Thanks, merging

**Pull request merged**